bundle-restricted/manifests/operator.ibm.com_namespacescopes.yaml

```diff
@@ -15,60 +15,74 @@ spec:
-          description: NamespaceScopeSpec defines the desired state of NamespaceScope
-          properties:
-            configmapName:
-              description: ConfigMap name that will contain the list of namespaces to be watched
-              type: string
-            manualManagement:
-              description: Set the following to true to manaually manage permissions for the NamespaceScope operator to extend control over other namespaces The operator may fail when trying to extend permissions to other namespaces, but the cluster administrator can correct this using the authorize-namespace command.
-              type: boolean
-            namespaceMembers:
-              description: Namespaces that are part of this scope
-              items:
-                type: string
-              type: array
-            restartLabels:
-              additionalProperties:
-                type: string
-              description: Restart pods with the following labels when the namspace list changes
-              type: object
-            serviceAccountMembers:
-              description: ServiceAccountMembers are extra service accounts will be bond the roles from other namespaces
-              items:
-                type: string
-              type: array
-          type: object
-        status:
-          description: NamespaceScopeStatus defines the observed state of NamespaceScope
-          properties:
-            validatedMembers:
-              description: 'INSERT ADDITIONAL STATUS FIELD - define observed state of cluster Important: Run "make" to regenerate code after modifying this file'
-              items:
-                type: string
-              type: array
-          type: object
-      type: object
-  version: v1
   versions:
   - name: v1
+    schema:
+      openAPIV3Schema:
+        description: NamespaceScope is the Schema for the namespacescopes API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: NamespaceScopeSpec defines the desired state of NamespaceScope
+            properties:
+              configmapName:
+                description: ConfigMap name that will contain the list of namespaces to be watched
+                type: string
```

Comment:
ConfigMap name will contain the **list** of namespaces, how the type is **string**? 
It will be like "ibm-common-services cp4i"? if those are the two namespaces watched.

Reply:
Yes, it is like "ibm-common-services,cp4i"